README.md

```diff
@@ -23,32 +23,15 @@ It also requires a valid `[API_KEY]`, which authenticates requests to the API. Y
 You can find your `[API_KEY]` and your `[HOST_IDENTIFIER]` within the [Credentials](https://app.swiftype.com/as/credentials) menu:
 
 ```ruby
-client = SwiftypeAppSearch::Client.new(:host_identifier => 'host-c5s2mj', :api_key => 'api-mu75psc5egt9ppzuycnc2mc3')
+client = SwiftypeAppSearch::Client.new(:account_host_key => 'host-c5s2mj', :api_key => 'private-mu75psc5egt9ppzuycnc2mc3')
 ```
 
 ### API Methods
 
 This client is a thin interface to the Swiftype App Search Api. Additional details for requests and responses can be
 found in the [documentation](https://swiftype.com/documentation/app-search).
 
-#### Indexing: Creating or Updating a Single Document
```

Comment:
Thanks for the submission @a17levine ! Indexing a single document exists, so this section should not be removed from the README - https://github.com/swiftype/swiftype-app-search-ruby/blob/master/lib/swiftype-app-search/client/documents.rb#L28

Reply:
@tehsven Hm interesting. It didn't show up when I did a `.methods` on the client object. I'll put it back

Reply:
Maybe its related to my environment